Formula/aptos.rb

```diff
@@ -0,0 +1,43 @@
+class Aptos < Formula
+  desc "Universal and fair access to decentralized assets for billions of people"
+  homepage "https://aptoslabs.com/"
+  url "https://github.com/aptos-labs/aptos-core/archive/refs/tags/aptos-cli-v1.0.3.tar.gz"
+  sha256 "670bb6cb841cb8a65294878af9a4f03d4cba2a598ab4550061fed3a4b1fe4e98"
+  license "Apache-2.0"
+
+  # This livecheck scans the releases folder and looks for all releases
+  # with matching regex of href="<URL>/tag/aptos-cli-v<SEMVER>". This
+  # is done to automatically check for new release versions of the CLI.
+  livecheck do
+    url "https://github.com/aptos-labs/aptos-core/releases"
+    strategy :page_match
+    regex(%r{href=["']?[^"' >]*?/tag/aptos-cli-v?(\d+(?:\.\d+)+)}i)
+  end
+
+  # Installs listed homebrew dependencies before aptos installation
+  # Dependencies needed: https://aptos.dev/cli-tools/build-aptos-cli
+  # See scripts/dev_setup.sh in aptos-core for more info
+  depends_on "cmake" => :build
+  depends_on "rustup-init" => :build
+  uses_from_macos "llvm" => :build
+
+  on_linux do
+    depends_on "pkg-config" => :build
+    depends_on "openssl@1.1"
+    depends_on "systemd"
+  end
+
+  # Currently must compile with the same rustc version specified in the
+  # root Cargo.toml file of aptos-core (currently it is v1.64).
```

Comment:
is there any open issue to support rust 1.66.0 (we dont see much differences between the two)

Reply:
Yes we are working to get it ready for 1.66, it just may take some time before its ready. Will update once complete.